src/community/Community.js

```diff
@@ -1,20 +1,72 @@
 import React, { Component } from 'react';
 import './Community.css';
-import SignupForm from '../shared/SignupForm';
+import {Card, CardMedia, CardTitle, CardText} from 'material-ui/Card';
+import SignupForm from '../shared/SignupForm'
+import sky from './sky.svg';
+import middleLandscape from './middle_landscape.svg';
+import deborah from './deborah.jpg';
+import eugene from './eugene.png';
+import sean from './sean.png';
 
 import DocumentTitle from 'react-document-title';
 
-class Community extends Component {
+export default class Community extends Component {
   render() {
+    const people = [
+      {
+        name: "Deborah Chang",
+        imageUrl: deborah,
+        buildCommunity: "I founded #NYCEDU to connect everyone with everyone in NYC education.",
+        getInvolved: <p>Subscribe to the newsletter to learn more.</p>
+      },
+      {
+        name: "Eugene Leventhal",
+        imageUrl: eugene,
+        buildCommunity: "I started the Bronx Education Meetup group to help connect those involved in education in the Bronx. I’m also working on EduDAO, a blockchain educational funding project.",
+        getInvolved: <p>Sign up for the <a href="https://www.meetup.com/Bronx-Education-Meetup/">meetup</a> or email me at <a href="mailto:eugene.leventhal@gmail.com">eugene.leventhal@gmail.com</a>.</p>
+      },
+      {
+        name: "Sean Perkins",
+        imageUrl: sean,
+        buildCommunity: "I started the Bronx Education Meetup group to help connect those involved in education in the Bronx. I’m also working on EduDAO, a blockchain educational funding project.",
```

Comment:
Build Community: I am working to ensure the technology we create for NYCEDU stays open and maintainable by the community. I also like to help folks that are new to software development get started in the field.